packages/legend-application-studio/src/components/editor/side-bar/LocalChanges.tsx

```diff
@@ -172,8 +173,11 @@ export const LocalChanges = observer(() => {
     editorStore.changeDetectionState.workspaceLocalLatestRevisionState.changes;
   const openChange =
```

Comment:
Here I think this is just because of current behaviour but i think now we can actually open a diff if they click on it and they are in text mode right? 
Either ways if openChange only works for form mode, we should think about disabling the button or some kind of UI indiactor?

Reply:
I don't think they can open diff editor when they are in text mode. It just doesn't open anything. We can diable the button. That would be a good option. This reminds me we need to disable `globalTestRunner` too. 
Will discuss more on that offline